### DISCUSSION TITLE
Checkout date importing via CSV

### DISCUSSION THREAD

**Discussion by mertokocum** [snipe/snipe-it#14716] [Q&A]
We are importing our assets without problem via CSV. We can also import checkouts with username and e-mail via CSV. But We cannot import the checkout date via CSV. Can you help us about this problem?

**Comment by snipe** [comment]
You'd likely need to use the History Importer if you want to fill in checkouts from back in time. https://snipe-it.readme.io/docs/importing-history

**Reply by mertokocum** [reply]
https://drive.google.com/file/d/15yA2aHdAnV8VYl_xhrmnEYIVojT-INfW/view?usp=sharing
nothing is changed. please note that my purpose is only to change the check-out date.
